team-config.yml

```diff
@@ -211,4 +211,84 @@ shared-image-gallery:
     build_notices_channel: "#platops-build-notices"
   tags:
     application: 'core'
-    businessArea: "Cross-Cutting"
\ No newline at end of file
+    businessArea: "Cross-Cutting"
+
+caps:
+  team: "Centralised Attachment of earnings Payment System"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "caps"
+    businessArea: "Cross-Cutting"
+
+pcol:
+  team: "Possession Claim On-Line"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "pcol"
+    businessArea: "Cross-Cutting"
+
+caseman:
+  team: "Case Management of County Courts"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "caseman"
+    businessArea: "Cross-Cutting"
+
+mcol:
+  team: "Money Claim On-Line"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "mcol"
+    businessArea: "Cross-Cutting"
+
+bais:
+  team: "BAU Integration Service"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "bais"
+    businessArea: "Cross-Cutting"
+
+darts-idam:
+  team: "darts-idam"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "darts-idam"
+    businessArea: "Cross-Cutting"
+
+sdt:
+  team: "Secure Data Transfer"
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "heritage-secure-data-transfer"
+    businessArea: "Cross-Cutting"
+
+template-management:
+  team: "template-management"
```

Comment:
Just checking why this and darts-idam don't follow the same pattern as a full team name?

Reply:
@Jenniferstrej updated Darts-idam as found this in Confluence.

Have left template-management as is, as that is what it should be according to information within Confluence spaces.